gdk/commands/component/PublishCommand.py

```diff
@@ -67,6 +101,32 @@ def upload_artifacts_s3(self, component_name, component_version):
         try:
             bucket = self.project_config["bucket"]
             region = self.project_config["region"]
+            encrypt = self.project_config.get("encrypt")
```

Comment:
We can move all of these newly added lines to another helper method.

Reply:
Fixed in new commit.